lib/auth/rotate.go

```diff
@@ -380,7 +379,7 @@ func (a *Server) autoRotate(ca types.CertAuthority) error {
 
 // processRotationRequest processes rotation request based on the target and
 // current phase and state.
-func processRotationRequest(req rotationReq) (types.CertAuthority, error) {
+func processRotationRequest(req rotationReq, keyStore keystore.KeyStore) (types.CertAuthority, error) {
```

Comment:
Nit: Would it make sense to just make this method and startNewRotation members of Server so you don't have to pass an additional function argument?

Reply:
good point, done in latest commit